pkg/binlog/event/common.go

```diff
@@ -50,7 +50,7 @@ func GenCommonFileHeader(flavor string, serverID uint32, gSet gtid.Set) ([]*repl
 			Flags:     defaultHeaderFlags,
 		}
 		latestPos   = uint32(len(replication.BinLogFileHeader))
-		prevGTIDsEv *replication.BinlogEvent // for MySQL, this will be a GenericEvent
+		prevGTIDsEv *replication.BinlogEvent
```

Comment:
is the new go-mysql change the rule？

Reply:
Yes, new go-mysql adds a new `PreviousGTIDsEvent ` struct to replace `GenericEvent with EventType=PREVIOUS_GTIDS_EVENT`.